SIT/test.sh

```diff
@@ -36,11 +35,11 @@ fi
 
 _captureOutput() {
   _info "Copying ${DOCKER_CDM}:/${testDir} into ${testDir}/output"
-  docker cp ${DOCKER_CDM}:/${testDir} ${testDir}/output
-  _info "Moving ${testDir}/output/$(basename ${testDir})/*.out TO ${testDir}/output"
-  mv ${testDir}/output/$(basename ${testDir})/*.out ${testDir}/output
-  _info "Moving ${testDir}/output/$(basename ${testDir})/*.err TO ${testDir}/output"
-  mv ${testDir}/output/$(basename ${testDir})/*.err ${testDir}/output
+  docker cp ${DOCKER_CDM}:/${testDir} ${testDir}/output/
+  _info "Moving ${testDir}/output/$(basename ${testDir})/output/*.out TO ${testDir}/output"
+  mv ${testDir}/output/$(basename ${testDir})/output/*.out ${testDir}/output/
```

Comment:
This line for example is resolving as,
```
INFO Moving smoke/05_reserved_keyword/output/05_reserved_keyword/*.err TO smoke/05_reserved_keyword/output
```
where, `smoke/05_reserved_keyword/output/05_reserved_keyword` directory doesn't exist today